web/src/aws-exports.js

```diff
@@ -1,13 +0,0 @@
-/* eslint-disable */
-// WARNING: DO NOT EDIT. This file is automatically generated by AWS Amplify. It will be overwritten.
-
-const awsmobile = {
-    "aws_project_region": "us-east-1",
-    "aws_appsync_graphqlEndpoint": "https://rtta7pxyhrdwrlg3d2vrw2mdri.appsync-api.us-east-1.amazonaws.com/graphql",
-    "aws_appsync_region": "us-east-1",
-    "aws_appsync_authenticationType": "API_KEY",
-    "aws_appsync_apiKey": "da2-bnjdiewpkjh2dl7y7dyz4kn7bu"
```

Comment:
Nice catch! : )

Would you mind to add this file to the root directory `.gitignore`? Looks like we accidentally merge this in which has some data that shouldn't exist in the git. We might need to do something like [this](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository). But with how there's a sprint1 branch still not merged into `dev`, might cause troubles for others

Reply:
I added this to git so that the other web developers wouldn't have to generate this locally with amplify. You're welcome to remove it, but we'll need to help them all get this set up locally.

Reply:
Good point, I can keep it in here then for now